src/coreComponents/physicsSolvers/simplePDE/PhaseFieldDamageFEM.cpp

```diff
@@ -247,10 +243,175 @@ void PhaseFieldDamageFEM::AssembleSystem( real64 const GEOSX_UNUSED_PARAM( time_
+                  //                                    Na * qp_damage * (1 + 2 * ell*strainEnergyDensity/Gc)) );
+
+
+                  element_rhs( a ) -= detJ[k][q] * (Na * qp_damage +
+                                                    (pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( qp_grad_damage, dNdX[k][q][a] ) +
+                                                     Na * constitutiveUpdate.GetDegradationDerivative( qp_damage ) * (ell*strainEnergyDensity/Gc)) );
+                }
+
+                for( localIndex b = 0; b < numNodesPerElement; ++b )
+                {
+                  real64 Nb = finiteElement->value( b, q );
+                  if( m_localDissipationOption == "Linear" )
+                  {
+                    // element_matrix( a, b ) -= detJ[k][q] *
+                    //                           (0.375*pow( ell, 2 ) * LvArray::tensorOps::AiBi<3>( dNdX[k][q][a], dNdX[k][q][b] ) +
+                    //                            (ell * D/Gc) * Na * Nb);
+                    //
+                    element_matrix( a, b ) -= detJ[k][q] *
+                                              (0.375*pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( dNdX[k][q][a], dNdX[k][q][b] ) +
+                                               (0.5 * ell * D/Gc) * constitutiveUpdate.GetDegradationSecondDerivative( qp_damage ) * Na * Nb);
+
+                  }
+                  else
+                  {
+                    // element_matrix( a, b ) -= detJ[k][q] *
+                    //                           ( pow( ell, 2 ) * LvArray::tensorOps::AiBi<3>( dNdX[k][q][a], dNdX[k][q][b] ) +
+                    //                               Na * Nb * (1 + 2 * ell*strainEnergyDensity/Gc )
+                    //                           );
+
+                    element_matrix( a, b ) -= detJ[k][q] *
+                                              ( pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( dNdX[k][q][a], dNdX[k][q][b] ) +
+                                                Na * Nb * (1 + constitutiveUpdate.GetDegradationSecondDerivative( qp_damage ) * ell*strainEnergyDensity/Gc )
+                                              );
+                  }
+                }
+              }
+            }
+            matrix.add( elemDofIndex, elemDofIndex, element_matrix );
+            rhs.add( elemDofIndex, element_rhs );
+          }
+        }
+      } );
+    } );
+  }
+  matrix.close();
+  rhs.close();
+
+  if( getLogLevel() == 2 )
+  {
+    GEOSX_LOG_RANK_0( "After PhaseFieldDamageFEM::AssembleSystem" );
+    GEOSX_LOG_RANK_0( "\nJacobian:\n" );
+    std::cout << matrix;
+    GEOSX_LOG_RANK_0( "\nResidual:\n" );
+    std::cout << rhs;
+  }
 
+  if( getLogLevel() >= 3 )
+  {
+    NonlinearSolverParameters & solverParams = getNonlinearSolverParameters();
+    integer newtonIter = solverParams.m_numNewtonIterations;
+#endif
 
```

Comment:
remove?

Reply:
Yeah, I think we left this here in case the PhaseFieldDamageKernels file didn't work, but it worked just fine.